cmd/immuadmin/command/database.go

```diff
@@ -102,9 +112,49 @@ func (cl *commandline) database(cmd *cobra.Command) {
 		},
 		Args: cobra.ExactArgs(1),
 	}
-	addDbUpdateFlags(cc)
+	addDbUpdateFlags(createCmd)
+
+	openCmd := &cobra.Command{
+		Use:               "open",
+		Short:             "Open database",
+		Example:           "open {database_name}",
+		PersistentPreRunE: cl.ConfigChain(cl.connect),
+		PersistentPostRun: cl.disconnect,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := cl.immuClient.OpenDatabase(cl.context, &schema.OpenDatabaseRequest{
```

Comment:
I think `open` / `close` naming could be misleading here - like in this context - I would assume that `OpenDatabase` is about connecting to given database.

Maybe load / unload ?